src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -74,10 +75,10 @@ class TranslateBehavior extends Behavior
         'fields' => [],
         'translationTable' => 'I18n',
         'defaultLocale' => '',
-        'model' => '',
+        'referenceName' => '',
+        'allowEmptyTranslations' => true,
         'onlyTranslated' => false,
```

Comment:
Should this be named `returnEmptyTranslations` and `returnOnlyTranslated`(or similar)?